R/artifacts.R

```diff
@@ -0,0 +1,346 @@
+#' @importFrom stringr str_extract
+generate_s3_key_bucket_ext <- function(artifact_name, run_id = get_active_run_id(), client = mlflow_client()) {
```

Comment:
does `_ext` signify extension? maybe `_info` is better given the list element that's returned